scm-webapp/src/main/java/sonia/scm/api/v2/resources/ResourceLinks.java

```diff
@@ -134,6 +136,10 @@ public String toInternal(String name) {
     public String publicKeys(String name) {
       return publicKeyLinkBuilder.method("findAll").parameters(name).href();
     }
+
+    public String apiKeys(String name) {
```

Comment:
If I read it correctly, this relates to the user's name, not the name of a key. I think we should make this clear everywhere we need to distinguish between the two.

Reply:
This kinda breaks the pattern since we use `name` for every method because it is referenced to the UserResources. But i still will change it because i like specific param names more.